packages/common/src/ui/components/menus/DropdownMenu/DropdownMenu.tsx

```diff
@@ -34,10 +34,9 @@ export const DropdownMenuItem: FC<DropdownMenuItemProps> = ({
       ) : null}
       <ListItemText
         inset={props.inset}
-        primaryTypographyProps={{ sx: { fontSize: 'inherit' } }}
+        primaryTypographyProps={{ color, sx: { fontSize: 'inherit' } }}
```

Comment:
leftover issue from [this comment](https://github.com/openmsupply/openmsupply-client/pull/170#discussion_r711423474)